MapboxDirections/Match/MBMatchOptions.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+
+@objc(MBMatchOptions)
+open class MatchOptions: RouteOptions {
+    public init(locations: [CLLocation], profileIdentifier: MBDirectionsProfileIdentifier? = nil) {
+        let waypoints = locations.map {
+            Waypoint(location: $0)
+        }
+        self.timestamps = locations.map { $0.timestamp }
+        super.init(waypoints: waypoints, profileIdentifier: profileIdentifier)
+    }
+    
+    @objc open var resample: Bool = false
+    
+    @objc open var timestamps: [Date]?
```

Comment:
Now that the timestamps are specified on the CLLocations in the input, make this property private.